0/substrate-contracts-node.md

```diff
@@ -0,0 +1,24 @@
+Running a Substrate Smart Contracts Node
+===
+
+After successfully installing [`substrate-contracts-node`](https://github.com/paritytech/substrate-contracts-node),
+you can start a local development chain by running:
+
+```bash
+substrate-contracts-node --dev --tmp
+```
+
+![Substrate Smart Contracts Node](assets/start-canvas-node.png)
```

Comment:
nit: This asset is outdated, it shows the `canvas` CLI

Reply:
Created a follow-up: https://github.com/substrate-developer-hub/substrate-contracts-workshop/issues/136.